cmd/csppserver/server.go

```diff
@@ -265,6 +270,9 @@ func setupTLS() (tc *tls.Config, selfsignedCert []byte) {
 			log.Fatal("certificate has no valid Subject Alternate Name")
 		}
 		tc.ServerName = x509Cert.DNSNames[0]
+		if len(x509Cert.DNSNames) > 1 {
+			alternateServerNames = append(alternateServerNames, x509Cert.DNSNames[1:]...)
+		}
```

Comment:
Instead of using append here, the slice can be set to all of the SANs (the DNSNames slice directly).  This func than then return all of the valid server names directly.

Reply:
Does it have to handle the case where there are multiple .onion SANs  (e.g. v2/v3), or just use the first .onion address in `DNSNames`?